Tests/TestData/spy_qqq_cov.csv

```diff
@@ -0,0 +1,629 @@
+QQQ,20200227,2109000,2143700,2055000,2056400,108461905,24256304147.220585
+SPY,20200228,2886200,2962600,2855400,2962600,346777226,
+QQQ,20200228,1986400,2071700,1981600,2058000,137166353,24048253485.10701
+SPY,20200302,2980000,3091500,2944600,3090900,207524129,
+QQQ,20200302,2087600,2165700,2059400,2164200,97482567,23952953131.758884
+SPY,20200303,3095500,3138000,2975700,3002400,276438674,
+QQQ,20200303,2170300,2196100,2076200,2094800,121621623,23772999001.92846
+SPY,20200304,3061100,3131000,3033300,3128600,150015051,
+QQQ,20200304,2133500,2183300,2113700,2182200,64764691,23700303988.490803
+SPY,20200305,3049500,3084700,3000200,3024600,162185198,
+QQQ,20200305,2125300,2163600,2101000,2115900,72037789,23502428545.81711
+SPY,20200306,2930800,2987800,2902400,2974600,201271746,
+QQQ,20200306,2045500,2091500,2029100,2080200,79862351,23232662517.864388
+SPY,20200309,2756000,2841400,2734600,2742300,280056192,
+QQQ,20200309,1930000,2011600,1921100,1935700,110141061,22995871113.640984
+SPY,20200310,2847600,2885200,2735000,2884200,248563890,
+QQQ,20200310,2011800,2043000,1936900,2041100,90119540,22763278574.590733
+SPY,20200311,2807300,2819100,2708900,2743600,232891616,
+QQQ,20200311,1993000,2007500,1927300,1952200,83550670,22597508965.88336
+SPY,20200312,2560500,2666400,2477200,2481100,347822422,
+QQQ,20200312,1820900,1907800,1769400,1773200,126249753,22845300517.92841
+SPY,20200313,2633400,2710600,2495800,2693200,287485966,
+QQQ,20200313,1870700,1944800,1781300,1923400,125012600,22740943884.462727
+SPY,20200316,2407800,2569000,2373700,2398500,266357640,
+QQQ,20200316,1741500,1847000,1691600,1693000,84060751,23256527570.98543
+SPY,20200317,2450400,2559900,2370800,2528000,235335708,
+QQQ,20200317,1753400,1843900,1697800,1821400,115793406,23395666471.73241
+SPY,20200318,2362500,2460300,2280800,2400000,300967068,
+QQQ,20200318,1715000,1792200,1668000,1766000,103468629,23745370015.49379
+SPY,20200319,2392600,2473800,2322500,2405100,263646203,
+QQQ,20200319,1754300,1834900,1716200,1776600,114691905,24064993681.78017
+SPY,20200320,2425900,2444600,2286200,2288000,304651791,
+QQQ,20200320,1818900,1828700,1701000,1707000,104841464,24706491960.728664
+SPY,20200323,2280000,2293900,2182700,2229500,281816983,
+QQQ,20200323,1707000,1742100,1649200,1704600,114873208,25350075441.092583
+SPY,20200324,2349900,2441000,2338000,2431500,214693586,
+QQQ,20200324,1798800,1838300,1779500,1836600,86434644,25484181906.50064
+SPY,20200325,2450000,2563500,2397600,2467900,278538590,
+QQQ,20200325,1844900,1895100,1799800,1823000,82965783,25650271902.86495
+SPY,20200326,2492600,2628000,2490500,2612000,224153673,
+QQQ,20200326,1839300,1927200,1835600,1919000,74995622,25571440846.139008
+SPY,20200327,2532200,2608000,2510500,2534200,193638954,
+QQQ,20200327,1869000,1905100,1846600,1853000,54182139,25638656906.974693
+SPY,20200330,2556800,2623900,2535300,2616500,141153243,
+QQQ,20200330,1870600,1924900,1863500,1920400,46667707,25591109173.308296
```

Comment:
These covariance values do not seem accurate? `25591109173.308296` the order of magnitude seems off. Believe the mean formular is off

Reply:
See my other comments https://github.com/QuantConnect/Lean/pull/7843#discussion_r1520050464